taqueria-vscode-extension-web-ui/src/data-editors/MichelineEditor.tsx

```diff
@@ -0,0 +1,228 @@
+import { emitMicheline, Parser } from '@taquito/michel-codec';
+import React, { useState } from 'react';
+import { WebviewApi } from 'vscode-webview';
```

Comment:
What the package is for? Does not look like something we can rely on 😕 
https://www.npmjs.com/package/vscode-webview

Reply:
This is actually: https://www.npmjs.com/package/@types/vscode-webview

Reply:
Ah cool, so it's only types definition for a package that exist in the runtime of vscode 👌